drivers/sqlite/sqlite.go

```diff
@@ -0,0 +1,321 @@
+// Initial code generated by generator.
+package sqlite
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/go-morph/morph/drivers"
+	"github.com/go-morph/morph/models"
+	_ "modernc.org/sqlite"
+)
+
+const driverName = "sqlite"
```

Comment:
nit for consistency

```suggestion
const driverName = "sqlite3"
```

Reply:
I went with the library name but not sure which is the right way of using the sqlite name :p @agnivade any thoughts?

Reply:
0/5 :stuck_out_tongue: